secure_message/resources/messages.py

```diff
@@ -75,7 +75,6 @@ def _alert_listeners(message):
         """used to alert user and case service once messages have been saved"""
```

Comment:
Remove case service from ` """used to alert user and case service once messages have been saved"""`